MotionMark/tests/resources/main.js

```diff
@@ -59,6 +59,9 @@ Sampler = Utilities.createClass(
     }
 });
 
+const sampleTimeIndex = 0;
+const sampleComplexityIndex = 1;
```

Comment:
I think we should have two different sets for the indices. One prefixed with `sample` and other prefixed with `processedSample`. So this set should include 

```
const sampleFrameLengthEstimate = 2;
```